internal/schema/cost_component.go

```diff
@@ -55,6 +56,14 @@ func (c *CostComponent) PriceHash() string {
 	return c.priceHash
 }
 
+func (c *CostComponent) SetCustomPrice(price *decimal.Decimal) {
+	c.customPrice = price
+}
+
+func (c *CostComponent) CustomPrice() *decimal.Decimal {
+	return c.customPrice
+}
+
```

Comment:
Personally, I don't see the point of these setters and getters. I would prefer just an exported `CustomPrice` field.

Reply:
I sort of agree they are pointless and it seems like a personal aesthetic preference either way, but I think it is worthwhile to be consistent with Price/PriceHash defined above.  If it's important to clean those up I guess we could have a tech debt task.